schema/information_schema_1-2-0.json

```diff
@@ -0,0 +1,241 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://Orange-OpenSource.github.io/its-client/schema/info",
+  "description": "Information JSon schema",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "type",
+    "version",
+    "instance_id",
+    "instance_type",
+    "running",
+    "timestamp",
+    "validity_duration"
+  ],
+  "properties": {
+    "type": {
+      "description": "type of server",
+      "type": "string",
+      "enum": [
+        "broker"
+      ]
+    },
+    "version": {
+      "type": "string",
+      "description": "json message format version",
+      "const": "1.1.99"
+    },
+    "instance_id": {
+      "description": "unique id all other the world for a server",
+      "type": "string"
+    },
+    "instance_type": {
+      "description": "type of instance",
+      "type": "string",
+      "enum": [
+        "edge",
+        "central"
+      ]
+    },
+    "central_instance_id": {
+      "description": "unique id all other the world for the central server if the instance is of type edge",
+      "type": "string"
+    },
+    "running": {
+      "description": "state of the server: true id running, false else.",
```

Comment:
if instead of id

Reply:
As explained in a previous comment: this commit just copies the 1.1.0 schema.